### DISCUSSION TITLE
Is there a command to focus on another window?

### DISCUSSION THREAD

**Discussion by Vamsee30** [qutebrowser/qutebrowser#5770] [Q&A]
I have two browser windows open side by side. Is there a command to change my focus from one to another? I know there is one for tabs, but what about windows?

**Comment by ninewise** [comment]
AFAIK this is left to the window manager. Try somthing like Alt or Super + Tab.

**Reply by martin-braun** [reply]
I think it would be nice if there was something like `tab-focus last` that works across windows. `qutebrowser` knows about the focus history, your window manager does not. We can already hop between windows using `tab-select` so I would expect it to also jump to its last window if its last tab is in a different window using `tab-focus last`.

Is my intuition flawed? @The-Compiler

EDIT: `tab-focus` is definitely the wrong one, because `tab-focus` will only care about tabs in the same window. `tab-select` will go across windows, but it won't allow to use `last` (which would be a cool addition). At the very least you can map keymaps to the leading indexes and switch between windows like that, but it will always jump to the very first tab, so I cannot simply do this in a 2 column setup:

```py
config.bind('<Ctrl-W>h', 'tab-select 0/')
config.bind('<Ctrl-W>l', 'tab-select 1/')
```

:/

**Reply by The-Compiler** [reply]
I added a comment to #5678 FWIW.

(Also please don't ping me for random comments, you can generally expect open source maintainers to be watching comments/notifications in their own repos, and I'd rather not be drowned in pings for things that aren't urgent)
